package.json

```diff
@@ -8,7 +8,7 @@
   ],
   "main": "index.js",
   "engines": {
-    "node": ">=8.9"
```

Comment:
Removing LTS version support should be marked as BREAKING CHANGE.
Refer loopback-next's recent node version commits for more insights.